react/components/InfoCard/index.js

```diff
@@ -122,20 +125,21 @@ const InfoCard = ({
     formatIOMessage({ id: mobileImageUrl, intl })
   )
 
-  const containerStyle = isFullModeStyle
+  const containerStyle = isFullModeStyle && !lazyLoad
     ? { backgroundImage: `url(${finalImageUrl})`, backgroundSize: 'cover' }
     : {}
 
   const containerClasses = classNames(
-    `${handles.infoCardContainer} items-center`,
+    `${handles.infoCardContainer} items-center relative`,
     {
       [`flex-ns ${flexOrderToken} bg-base ph2-ns pb2 justify-between`]: !isFullModeStyle,
       [`bg-center bb b--muted-4 flex ${justifyToken}`]: isFullModeStyle,
+      'lazysizes': lazyLoad,
```

Comment:
```suggestion
      lazysizes: lazyLoad,
```

Reply:
since this is for class names, I like it better as a quoted string. could you let this slide please?